esda/lee.py

```diff
@@ -0,0 +1,172 @@
+import numpy
+from scipy import sparse
+from sklearn.base import BaseEstimator
+from sklearn import preprocessing
+from sklearn import utils
+from libpysal.weights import lag_spatial, WSP
+
+def _lee_from_R(x,y,W, local, zero_policy=None, NAOK=False):
+    from rpy2.robjects.packages import importr
+    from rpy2.robjects import r as R, numpy2ri
+    from rpy2.rinterface import NULL
+
+    zero_policy = NULL if zero_policy is None else zero_policy
+
+    spdep = importr('spdep')
```

Comment:
Is `importr` new?

Reply:
would this make spdep a hard dependency?

Reply:
I think the r code is slated for removal. If so, these comments can be resolved. Just wanted to flag it.